src/components/Tabs/Tab.tsx

```diff
@@ -0,0 +1,176 @@
+import styled from '@emotion/styled'
+import {
+  ComponentProps,
+  ElementType,
+  ForwardedRef,
+  KeyboardEventHandler,
+  MouseEventHandler,
+  ReactNode,
+  Ref,
+  forwardRef,
+  useMemo,
+} from 'react'
+import Tooltip from '../Tooltip'
+import { useTabsContext } from './TabsContext'
+
+const StyledCounter = styled.span`
+  border-radius: 50%;
+  font-size: 12px;
+  background-color: ${({ theme }) => theme.colors.neutral.backgroundStrong};
+  border: 1px solid ${({ theme }) => theme.colors.neutral.borderStrong};
+  margin-left: ${({ theme }) => theme.space['1']};
+  height: 24px;
+  width: 24px;
+  text-overflow: ellipsis;
+  line-height: 22px;
+`
```

Comment:
It couldn't work to use component `Text` ? So we can remove all props linked to text